packages/safe-apps-provider/src/provider.ts

```diff
@@ -106,7 +125,8 @@ export class SafeAppProvider implements AsyncSendable {
           const resp = await this.sdk.txs.getBySafeTxHash(txHash);
           txHash = resp.transactionHash || txHash;
         } catch (e) {}
-        return this.sdk.eth.getTransactionReceipt([txHash]).then((tx) => {
+        // eslint-disable-next-line
+        return this.sdk.eth.getTransactionReceipt([txHash]).then((tx: any) => {
```

Comment:
same here

Reply:
it was like that before ... not sure why it did not complain

Reply:
<img width="839" alt="Screenshot 2021-03-29 at 13 08 53" src="https://user-images.githubusercontent.com/16622558/112814110-fb8d5a00-908f-11eb-8acb-6e62e2d5e85e.png">

for me the type is inherited and it should be like that

Reply:
interesting 🤔  ... anyways I set it now ... so it is correct in any way

Reply:
actually it does for me too now ... ok I will remove the import ... not sure what vscode did there